### PR TITLE
livestream page

### PR DESCRIPTION
**Pull request by LeonardTarigan**

### CODE REVIEW

File: components/shared/BetSlipCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import { Listbox, Transition } from '@headlessui/react';
+import { useState, Fragment } from 'react';
+
+const bids = [5, 10, 20, 30, 50, 100];
+
+function BetSlipCard() {
+  const [selected, setSelected] = useState<number>(bids[0]);
+
+  return (
+    <div className="w-full bg-xport-gray-alternate text-xs rounded">
```
Comment (galihpermana29) [inline components/shared/BetSlipCard/index.tsx]: this should be width full, by my view this is not full width??? enough??
Screen Shot 2023-09-20 at 18 00 15

also the payout shouldbe automatically calculated if the dropdown is change... by the calculation payout = dropdown value * odds

File: components/shared/BetSlipCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        <div className="flex gap-2 items-center">
+          <div className="aspect-square w-5 basis-[15%] rounded-full bg-xport-black-light animate-pulse"></div>
+          <span className="font-semibold text-xport-game-mlbb">
+            Evos Esports vs RRQ Esports
+          </span>
+        </div>
+        <div className="flex justify-between items-center font-semibold">
+          <div className="flex items-center gap-2">
+            <div className="h-5 w-5 rounded-full bg-xport-black-light animate-pulse"></div>
+            <span>Evos Esport</span>
```
Comment (galihpermana29) [inline components/shared/BetSlipCard/index.tsx]: this name should be dynamic regarding what's user click on this button 
Screen Shot 2023-09-20 at 18 02 11

File: app/livestream/page.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      </section>
+      <section className="md:basis-[30%] lg:basis-[20%] flex flex-col justify-between bg-xport-black-primary h-[50vh] md:h-[90vh] border-l border-b border-xport-gray-primary">
+        <div className="py-5 px-3">
+          <h2 className="font-semibold text-center mb-5">Bet Slip</h2>
+          <BetSlipCard />
+        </div>
+        <div className="bg-xport-orange-primary text-xs flex items-center">
+          <div className="bg-xport-gray-alternate w-2/3 flex flex-col gap2 pl-3 pr-4 py-5 rounded-r-full">
+            <div className="flex justify-between">
+              <span>Stake</span>
```
Comment (galihpermana29) [inline app/livestream/page.tsx]: the stake and the payout should be dynamic, regading the changed that dropdown h

as been made

Comment (galihpermana29) [reply app/livestream/page.tsx]: Screen Shot 2023-09-20 at 18 03 26
also on that screen, the active bet should be related on whats button the 

user click in this ui

File: components/shared/BetSlipCard/index.tsx (hunk 1 above)
Comment (LeonardTarigan) [reply components/shared/BetSlipCard/index.tsx]: I added max-w-screen-xl which is around 1280px to maintain the ratio on bigger screen. Should I make it to 2xl (1536px) or just remove the max width?

**Pull request merged**